tests/unit_tests/sql_parse_tests.py

```diff
@@ -1938,36 +1938,40 @@ def test_sqlstatement() -> None:
     ],
 )
 @pytest.mark.parametrize(
-    "macro",
-    [
-        "latest_partition('foo.bar')",
-        "latest_partition(' foo.bar ')",  # Non-atypical user error which works
-        "latest_partition('foo.%s'|format('bar'))",
-        "latest_sub_partition('foo.bar', baz='qux')",
-    ],
-)
-@pytest.mark.parametrize(
-    "sql,expected",
+    "macro,expected",
     [
         (
-            "SELECT '{{{{ {engine}.{macro} }}}}'",
+            "latest_partition('foo.bar')",
+            {Table(table="bar", schema="foo")},
+        ),
+        (
+            "latest_partition(' foo.bar ')",  # Non-atypical user error which works
             {Table(table="bar", schema="foo")},
         ),
         (
-            "SELECT * FROM foo.baz WHERE quux = '{{{{ {engine}.{macro} }}}}'",
```

Comment:
This example no longer exists. It's a trivial extension from the previous logic, but was making the nested parametrization challenging.